src/ray/common/ray_config_def.h

```diff
@@ -445,8 +445,8 @@ RAY_CONFIG(std::string, custom_unit_instance_resources, "")
 // Maximum size of the batches when broadcasting resources to raylet.
 RAY_CONFIG(uint64_t, resource_broadcast_batch_size, 512);
 
-// If enabled and worker stated in container, the container will add
-// resource limit.
+/// If enabled, workers will add resource limits.
+/// This requires the raylet process to have permissions to create cgroups.
 RAY_CONFIG(bool, worker_resource_limits_enabled, false)
```

Comment:
Can you crash ray if this flag is true & cgroup is not available?

Reply:
I have add validation in node.py, PTAL